pontoon/base/models.py

```diff
@@ -1627,6 +1631,21 @@ def with_status_counts(self, locale):
                     default=0
                 )
             ),
+            rejected_count=Sum(
+                Case(
+                    When(
+                        Q(
+                            translation__locale=locale,
+                            translation__approved=False,
+                            translation__fuzzy=False,
+                            translation__rejected=True,
```

Comment:
Considering that when ``rejected`` is true, none of ``approved`` or ``fuzzy`` can be true, I think you can avoid the previous 2 filters. Not sure how big an impact on performance that would have though.